src/ExpressionEvaluator/Core/Source/ResultProvider/ResultProvider.cs

```diff
@@ -580,6 +563,15 @@ private static string GetTypeName(DkmInspectionContext inspectionContext, DkmClr
                         category: DkmEvaluationResultCategory.Other,
                         flags: value.EvalFlags,
                         evalFlags: inspectionContext.EvaluationFlags);
+
+                    if (dataItem.Kind != ExpansionKind.ExplicitEvaluationResult)
```

Comment:
Why this check if C++ is overriding the method?  Is it then calling the base implementation?